src/app/public/modules/summary-action-bar/summary-action-bar.component.html

```diff
@@ -32,36 +32,34 @@
         <div *ngIf="isSummaryCollapsible && !isSummaryCollapsed"
           class="sky-summary-action-bar-details-collapse"
         >
-          <button
-            class="sky-btn sky-btn-secondary"
-            type="button"
+          <sky-chevron
             [attr.title]="'skyux_summary_action_bar_hide_summary' | skyLibResources"
             [attr.aria-controls]="summaryId"
             [attr.aria-expanded]="true"
-            (click)="hideSummarySection()"
+            [direction]="isSummaryCollapsed ? 'up' : 'down'"
```

Comment:
I don't think we need both `sky-chevron` components now do we? Can't we just use one with the `direction` property? Or am I forgetting some nuance here?

Reply:
The differing nesting structure of the template based on what mode you're in makes that tricky. To clean things up I put the chevron inside an `ng-template` is that better?